src/types/shared.ts

```diff
@@ -26,6 +28,13 @@ export enum AccountFlags {
   CREATE_ORGANIZATIONS = "create_organizations",
 }
 
+export const DEFAULT_INDIVIDUAL_FLAGS: AccountFlags[] = [
+  AccountFlags.CREATE_REPOSITORIES,
+  AccountFlags.CREATE_ORGANIZATIONS,
+];
```

Comment:
```suggestion
export const DEFAULT_INDIVIDUAL_FLAGS: AccountFlags[] = [];
```

@jedsundwall Confirming that we don't want users to be able to create orgs or repos by default?

Reply:
Yes this is correct. We need to grant them permission.